package book.chapters.two;

public class Excep4 {

    public void print4() {

        System.out.println("In Exception4");
        throw new RuntimeException();
    }
}
